src/components/sidebar/Sidebar.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../utilis/colors';
+@import '../../utilis/mixins';
+
+.sidebar-container {
+  position: absolute;
+  left: 0;
+  top: 0;
+
+  background: linear-gradient(179.32deg, $background-gradient-first-color 8.04%, $background-gradient-second-color 99.41%);
+  border-right: 1px solid $border-color;
+
+  height: 100vh;
+
+  padding: 30px;
+
+  display: flex;
+  flex-direction: column;
+}
+
+.logo-section {
+  margin-bottom: 30px;
+}
+
+.profile-picture-section {
+  width: 54px;
+  height: 54px;
+}
+
+.user-profile-overview {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  border-radius: 6px;
+  padding: 10px;
+
+  border: 1px solid $border-color;
+}
+
+.user-name {
+  margin-bottom: 5px;
+
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 500;
+  color: #242834;
+}
+
+.user-role {
+  @include font("Inter", normal, 400, 14px, 18px, $text-color-primary);
+}
+
+.user-info-section * {
+  margin: 0;
+  padding: 0;
+}
+
+.user-info-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+
+  margin-left: 15px;
+  margin-right: 60px;
+}
+
+.expand-user-info-section {
+  margin-right: 10px;
+}
+
+
+.sidebar-options {
+  display: flex;
+  flex-direction: column;
+
+  justify-content: flex-start;
+
+  margin-top: 20px
+}
+
+.sidebar-option {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.sidebar-option p {
+  margin-left: 20px;
+
+  @include font("Gilroy", normal, 500, 14px, 16px, $text-color-secondary);
```

Comment:
I don't mean this. create mixin for similar staff for example font-family, font-weight

@mixin bold-font {
  font-family: 'Goldplay-Bold';
  font-style: normal;
  font-weight: 600;
} 
as you can se here I use just this 3 stuff and font size I add manualy